caas/kubernetes/provider/bootstrap.go

```diff
@@ -1479,6 +1569,7 @@ func (c *controllerStack) buildContainerSpecForCommands(jujudCmds []string) (*co
 	if err != nil {
 		return nil, errors.Annotate(err, "creating controller pod spec")
 	}
+	//spec.Containers[0].Env[0].Value += ",api-server"
```

Comment:
We need a better way to pass this to the controller charm container.

Reply:
yeah, why do we actually need this? can't remember

Reply:
ah it gets passed to the uniter. Looks like it's only used for the "Pebble" poller. Depends whether we need this for the api-server